ios/RNSQIPCardEntry.m

```diff
@@ -162,6 +162,111 @@ - (dispatch_queue_t)methodQueue
     });
 }
 
+RCT_REMAP_METHOD(startBuyerVerificationFlow,
+                 paymentSourceId
+                 : (NSString *)paymentSourceId
+                 locationId
+                 : (NSString *)locationId
+                 buyerActionString
+                 : (NSString *)buyerActionString
+                 moneyMap
+                 : (NSDictionary *)moneyMap
+                 contactMap
+                 : (NSDictionary *)contactMap
+                 startCardEntryFlowWithResolver
+                 : (RCTPromiseResolveBlock)resolve
+                     rejecter
+                 : (RCTPromiseRejectBlock)reject)
+{
+    dispatch_async([self methodQueue], ^{
+        SQIPMoney *money = [[SQIPMoney alloc] initWithAmount:[moneyMap[@"amount"] longValue]
+                            currency:[RNSQIPBuyerVerification currencyForCurrencyCode:moneyMap[@"currencyCode"]]];
+
+        SQIPBuyerAction *buyerAction = nil;
+        if ([@"Store" isEqualToString:buyerActionString]) {
+            buyerAction = [SQIPBuyerAction storeAction];
+        } else {
+            buyerAction = [SQIPBuyerAction chargeActionWithMoney:money];
+        }
+
+        SQIPContact *contact = [[SQIPContact alloc] init];
```

Comment:
Similar to android, can we extra the code to a method like this one?  https://github.com/square/in-app-payments-flutter-plugin/blob/f847432a6128f80e12d953d8fb46b4a48b5854a3/ios/Classes/FSQIPCardEntry.m#L76